extras/kms/testing_test.go

```diff
@@ -16,3 +18,58 @@ func Test_stub(t *testing.T) {
 	t.Log("rows: ", rows)
 	require.NoError(t, err)
 }
+
+func Test_TestDeleteKeyPurpose_TestDeleteAllKeys(t *testing.T) {
```

Comment:
Is there a reason to combine these as a single test instead of having two tests? I would find it easier to see the setup, execution, and assertions for each test function if they were separate tests.

Reply:
most of it is setup and it's like an additional 10 lines to test TestDeleteAllKeys after that setup.. just didn't make sense to me to split them.

Reply:
The extra comments help :+1: